python/ray/autoscaler/v2/tests/util.py

```diff
@@ -15,6 +15,13 @@ def make_autoscaler_instance(
     ray_node: Optional[autoscaler_pb2.NodeState] = None,
     cloud_instance_id: Optional[str] = None,
 ) -> AutoscalerInstance:
+
+    if cloud_instance_id:
+        if im_instance:
+            im_instance.cloud_instance_id = cloud_instance_id
+        if ray_node:
+            ray_node.instance_id = cloud_instance_id
```

Comment:
Do they represent the same thing? If so, using a consistent naming style would be better (i.e. use `ray_node.cloud_instance_id` instead of `ray_node.instance_id`.)

Reply:
yeah, unfortunately we had it named "instance_id" on ray's proto. I will prob leave it as a todo for the renaming in another PR.